osc-server/src/main/java/org/osc/core/broker/service/persistence/VirtualizationConnectorEntityMgr.java

```diff
@@ -101,6 +109,12 @@ public static void fromEntity(VirtualizationConnector vc, VirtualizationConnecto
         dto.setAdminProjectName(vc.getProviderAdminProjectName());
         dto.setAdminDomainId(vc.getAdminDomainId());
         dto.getProviderAttributes().putAll(vc.getProviderAttributes());
+
+        // For rabbit MQ password, decrypt it before setting it on the dto.
+        String rabbitMqPassword = dto.getProviderAttributes().get(ATTRIBUTE_KEY_RABBITMQ_USER_PASSWORD);
```

Comment:
This will not be exposed through the rest api right?

Reply:
yes, it will not be exposed in the API.
The following call as part of the list and get services will remove the password from the dto
org.osc.core.broker.service.dto.VirtualizationConnectorDto.sanitizeVirtualizationConnector(VirtualizationConnectorDto)